src/lib/liquidiumServer.test.ts

```
@@ -0,0 +1,92 @@
+
+    const expected: LiquidiumFetchResult<unknown> = {
+      ok: false,
+      message: 'Test: BAD',
+      details: '{"error":"BAD"}',
+      status: 400,
+    };
+    expect(result).toEqual(expected);
+  });
+
+  it('handles missing LIQUIDIUM_API_KEY', async () => {
+    delete process.env.LIQUIDIUM_API_KEY;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)